### DISCUSSION TITLE
Lab Assignment #2 👩‍💻

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/24SU-ASIAAM-191A#37] [Labs]
## Due 7/10

Please use this thread to post your [lab assignment's #2](https://albertkun.github.io/24SU-ASIAAM-191A/assignments/week2/lab_assignment/) link, which is the GitHub pages URL. [Click here for a reminder on how to get it](https://albertkun.github.io/24SU-ASIAAM-191A/labs/week1/3/#using-github-pages).

### Note 
If your map is not on the main `index.html` page, be sure to submit that sub-page's link instead:

#### For example:
https://albertkun.github.io/24SU-asiaam-191a-sample/week2/map.html

If you only have one page, then you can just submit the `index.html` link as per usual.

**Comment by stinest** [comment]
https://stinest.github.io/24su-191a-labs/week2/index.html

**Comment by sjeyakum** [comment]
https://sjeyakum.github.io/24su-aas191a-labs/week2/index.html

**Comment by pwinnie2026** [comment]
https://pwinnie2026.github.io/24su-asiaam-191a-labs/Week2Extra/index.html

**Comment by kianaptran** [comment]
https://kianaptran.github.io/24su-asiaam-191a-labs/week2submission/

**Comment by kimada62** [comment]
https://kimada62.github.io/24su-asiaam-191a-labs/week2

**Comment by Keiya-Wada** [comment]
https://keiya-wada.github.io/asiaam-labs/week2/index.html

**Comment by chrissanrow** [comment]
https://chrissanrow.github.io/asiaam191a-su24/week2/

**Comment by kchhoeu** [comment]
https://kchhoeu.github.io/24su-asiaam191a-labs/week2/

**Comment by jasminereyz** [comment]
https://jasminereyz.github.io/AA191-Labs/week2/

**Comment by giaari15** [comment]
https://giaari15.github.io/aas191-labs/week_2/index.html

**Reply by albertkun** [reply]
i don't think you have the right url??

**Reply by giaari15** [reply]
i think it works now! smth was wrong with the path for some reason >.<

**Reply by albertkun** [reply]
ok, it does work now! 👍

**Comment by NellieNguyen149** [comment]
https://nellienguyen149.github.io/24su-asianam-191a-labs/week2/index.html

**Comment by nesslee123** [comment]
https://nesslee123.github.io/Assignments/week2/index.html

**Comment by caitlynn-hauw** [comment]
https://caitlynn-hauw.github.io/aa191-assignments/week2/index.html

**Comment by zhamic7** [comment]
https://zhamic7.github.io/aa191a-labs/week2/index.html
